src/templates/ObservabilityPackDetails.js

```diff
@@ -158,7 +157,7 @@ const emptyStateContent = (pack, tabName) => {
         <Button
           as={Link}
           variant={Button.VARIANT.PRIMARY}
-          to={getPackUrl(pack.logoUrl)}
+          to={pack.packUrl}
```

Comment:
It looks like `packUrl` [is optional](https://source.datanerd.us/open-source/open-install-library-service/blob/master/src/graphql.schema.ts#L168). We might want to supply a fallback (e.g. the url for the quickstart repository) in the event that this url doesn't exist:

```js
// if it doesn't already exist as a constant...
const REPO_URL = "https://github.com/newrelic/newrelic-observability-packs";

const packUrl = pack.packUrl || REPO_URL;

// ...
to={packUrl}
```

Reply:
cool ye good to have a failsafe, would this also work:

```js
const { packUrl = REPO_URL } = pack;
```

Reply:
i think it's probably a little less clear but just out of interest